templates/consul.sysv.erb

```diff
@@ -52,7 +52,7 @@ start() {
         mkrundir
         [ -f $PID_FILE ] && rm $PID_FILE
         daemon --user=<%= scope.lookupvar('consul::user') %> \
-            --pidfile="$PID_FILE" \
+            --pidfile="$PID_FILE" --umask=<%= scope.lookupvar('consul::umask') %> \
```

Comment:
This breaks CentOS init scripts with the following error in the logs:

`/etc/init.d/consul: Usage: daemon [+/-nicelevel] {program} [FAILED]`